components/support/utils/src/test/java/mozilla/components/support/utils/URLStringUtilsTest.kt

```diff
@@ -160,6 +164,10 @@ class URLStringUtilsTest {
         assertTrue(isSearchTerm("http://www-.com"))
         assertTrue(isSearchTerm("www.c-c-  "))
         assertTrue(isSearchTerm("3-3 "))
+
+        // Examples from issues
+        assertFalse(isSearchTerm(
+            "https://disposable-wide4--solidsoft-blog-disposable.netlify.app/")) // fenix/10747
```

Comment:
Let's remove the fenix ticket references. I'd use a generic example and link to the A-C issue.